.circleci/config.yml

```diff
@@ -55,25 +23,87 @@ jobs:
             ./architect deploy
           fi
 
+
+
   e2eTestCurrentVersionPR:
     environment:
+      MINIKUBE_VERSION: v0.25.0
       TESTED_VERSION: "current"
```

Comment:
I had to make these blocks explicit because I could not find a way to get the reusable blocks notice the env vars. Maybe I am doing something wrong or miss the key part here. Feedback welcome.